public/index.html

```diff
@@ -33,6 +33,14 @@
     <script defer src="js/home.js"></script>
   </head>
   <body>
+    <header>
+      <div id="navigationBar"></div>
+    </header>
+
+    <script>
+      $('#navigationBar').load('./components/navbar.html')
```

Comment:
I was thinking we move this code to a script. Call it something like `common.js`. So like store it in `public/js/common.js`.

And the script does `$('header').load('./components/navbar.html');`

That way all our pages could include a script tag for `common.js` and it just auto inserts the navbar. The only each page needs is a `<script>` tag including `common.js` and it should have an empty `<header></header>`.

Also if we did that I think we could remove the `#navigationBar` div inside. We wouldn't need it.